src/icons/instagram.tsx

```diff
@@ -1,31 +1,10 @@
-import { FC } from 'react';
+import instagram from '@/assets/svg/instagram.svg';
+import Image from '@/features/image';
 
-export const InstagramIcon: FC = () => (
-  <svg xmlns="http://www.w3.org/2000/svg" width="57" height="57">
-    <rect x="0.666504" y="0.0683594" width="56" height="56" rx="28" fill="#EEEEEE" />
-    <path
-      d="M 33.166 18.068 L 23.166 18.068 C 20.405 18.068 18.166 20.307 18.166 23.068 L 18.166 33.068 C 18.166 35.83 20.405 38.068 23.166 38.068 L 33.166 38.068 C 35.928 38.068 38.166 35.83 38.166 33.068 L 38.166 23.068 C 38.166 20.307 35.928 18.068 33.166 18.068 Z"
-      fill="transparent"
-      strokeWidth="2"
-      stroke="rgb(0,0,0)"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-      strokeDasharray=""></path>
-    <path
-      d="M 32.167 27.438 C 32.29 28.271 32.148 29.121 31.761 29.867 C 31.373 30.614 30.76 31.22 30.008 31.598 C 29.257 31.976 28.405 32.108 27.575 31.974 C 26.744 31.841 25.977 31.448 25.381 30.854 C 24.787 30.259 24.394 29.491 24.261 28.661 C 24.127 27.83 24.259 26.978 24.637 26.227 C 25.015 25.475 25.621 24.862 26.368 24.475 C 27.114 24.087 27.965 23.945 28.797 24.068 C 29.646 24.194 30.432 24.59 31.038 25.197 C 31.645 25.804 32.041 26.59 32.167 27.438 Z"
-      fill="transparent"
-      strokeWidth="2"
-      stroke="rgb(0,0,0)"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-      strokeDasharray=""></path>
-    <path
-      d="M 33.666 22.568 L 33.676 22.568"
-      fill="transparent"
-      strokeWidth="2"
-      stroke="rgb(0,0,0)"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-      strokeDasharray=""></path>
-  </svg>
-);
+export const InstagramIcon = () => {
+  return (
+    <div className="icon-surface">
```

Comment:
[We decided](https://github.com/rolling-scopes/site/blob/main/readme/semantic-webaccessibility.md) not to use `div`s, so can we use `figure` here?

Reply:
Done